app/main.development.js

```diff
@@ -337,13 +417,22 @@ ipcMain.on("grpc-versions-determined", (event, versions) => {
   grpcVersions = { ...grpcVersions, ...versions };
 });
 
-const launchDCRD = () => {
+const launchDCRD = (credentials) => {
   var spawn = require("child_process").spawn;
-  var args = ["--configfile="+dcrdCfg()];
+  let args = [];
+
+  if(credentials){
+    const {rpcappdata} = credentials;
+    const rpccert = `--rpccert=${rpcappdata}/rpc.cert`;
```

Comment:
you can remove this, setting appdata then makes dcrd use the .cert/.key found in the specified directory.